CHANGELOG.md

```diff
@@ -0,0 +1,18 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+Each new release typically also includes the latest modulesync defaults.
+These should not affect the functionality of the module.
+
+## [Unreleased](https://github.com/voxpupuli/puppet-hashi_stack/tree/HEAD)
```

Comment:
can you set `version` in `metadata.json` to `1.0.0` and regenerate the changlog? That should replace the `Unreleased` with `1.0.0`

Reply:
done!